lexis/render/histogram.fbs

```diff
@@ -1,18 +1,21 @@
 // Copyright (c) 2016, Human Brain Project
 //                     Ahmet Bilgili <ahmet.bilgili@epfl.ch>
 
-// This event is used to communicate histograms for
-// 8 bit data between applications ( also inside the
-// application ).
+// This event is used to communicate histograms with
+// arbitrary number of bins.
 //
-// Currently every data is discretized to 256 bins.
-// This may change later to higer number of bins if
-// needed.
+// Data values between max and min are discretized into
+// number of the bins.
```

Comment:
A histogram divides a range [min, max] of possible values in n bins and counts how many elements fall inside each bin.